src/update/orderbook_rewards.py

```diff
@@ -0,0 +1,50 @@
+"""Pushes a new dune user generated view per Accounting Period
+with name dune_user_generated.cow_rewards_{{PeriodHash}}
```

Comment:
> Because appending to an existing view is more complicated than creating a new one. We could go this route, but if anyone ever drops the table it will be a real issue to reproduce

This makes more sense. Thanks for sharing. :+1: